pyproject.toml

```
@@ -6,9 +6,10 @@ requires = [
     "wheel",
     "Cython>=0.29.21,<3",  # Note: sync with setup.py
     "numpy==1.16.5; python_version=='3.7' and platform_system!='AIX'",
-    "numpy==1.17.3; python_version>='3.8' and platform_system!='AIX'",
+    "numpy==1.17.3; python_version=='3.8' and platform_system!='AIX'",
     "numpy==1.16.5; python_version=='3.7' and platform_system=='AIX'",
-    "numpy==1.17.3; python_version>='3.8' and platform_system=='AIX'",
+    "numpy==1.17.3; python_version=='3.8' and platform_system=='AIX'",
+    "numpy==1.19.2; python_version>='3.9'",
```

cc @TomAugspurger is this what we want here?

---

We just copy scipy: https://github.com/scipy/scipy/blob/37b650e04f3bf49bc11cfcd18f2848ad4f957a0d/pyproject.toml

```suggestion
    "numpy; python_version>='3.9'",
```

I'm not sure if scipy will update theirs once NumPy has a release that officially supports 3.9

---

ahh ok

---

ok @fangchenli i think you had this right before